src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/RetrieveOrganisationsTest.java

```diff
@@ -599,6 +638,35 @@ public void persists_and_returns_all_organisations_details_by_pending_and_active
         assertThat(orgResponse.get("http_status").toString().contains("OK"));
     }
 
+    @Test
+    public void persists_and_returns_all_organisations_details_by_pending_and_active_status_since()
+            throws InterruptedException {
+
+        String organisationIdentifier = createOrganisationRequest("PENDING");
+        String organisationIdentifier1 = createAndActivateOrganisationWithGivenRequest(
+                someMinimalOrganisationRequest().status("ACTIVE").sraId(randomAlphabetic(10)).build());
+
+        assertThat(organisationIdentifier).isNotEmpty();
+        assertThat(organisationIdentifier1).isNotEmpty();
+
+        TimeUnit.SECONDS.sleep(SINCE_PAUSE_SECONDS);
+        final LocalDateTime sinceValue = LocalDateTime.now();
+        final String since = sinceValue.format(DATE_TIME_FORMATTER);
+
+        String organisationIdentifier2 = createAndActivateOrganisationWithGivenRequest(
+                someMinimalOrganisationRequest().status("ACTIVE").sraId(randomAlphabetic(10)).build());
+
+        Map<String, Object> orgResponse = professionalReferenceDataClient
+                .retrieveAllOrganisationDetailsByStatusSinceTest("PENDING,ACTIVE", hmctsAdmin, since);
+
+        assertThat(orgResponse.get("organisations")).isNotNull();
+        assertThat(orgResponse.get("organisations")).asList().isNotEmpty();
+        assertThat(orgResponse.get("organisations")).asList().size().isEqualTo(1);
+        assertThat(orgResponse.get("organisations").toString()).contains("status=ACTIVE");
+
+        assertThat(orgResponse.get("http_status").toString().contains("OK"));

```

Comment:
```suggestion
        assertThat(orgResponse.get("http_status").toString()).contains("OK");
```

Reply:
with current format of brackets even something like

`assertThat(orgResponse.get("http_status").toString().contains("123"));` - passes